### DISCUSSION TITLE
Which branch to be on: master or release

### DISCUSSION THREAD

**Discussion by oren0e** [neoclide/coc.nvim#4921] [Q&A] [answered]
I'm using coc.nvim with vim, and Vundle plugin manager. Occasionally I update the plugins which is essentially doing `git pull` to all the plugins' repos. On coc.nvim I see that I'm on `release` branch. The last release was like almost 2 years ago, so my question is - should I be on the release branch or master branch in order to use the most updated plugin version for coc.nvim? (the branch on which I'm supposed to run `yarn install`)

**Comment by fannheyward** [comment, marked as answer]
1. It's recommended to use the `release` branch, this branch will be compiled, you don't need to do `npm install`
2. the `master` branch is the HEAD branch that fixes/features will be merged into master, you need to run `npm install && npm build` to compile it. This branch will wait for a time then build to `release` branch
3. the tag release, as you saw `The last release was like almost 2 years ago`, was outdated

> should I be on the release branch or master branch in order to use the most updated plugin version for coc.nvim

Yes, use `release` branch is recommended.

**Reply by oren0e** [reply]
Thank you!
